generator/src/lib.rs

```diff
@@ -1200,7 +1200,7 @@ pub fn generate_extension_commands<'a>(
 
     let ident = format_ident!(
         "{}Fn",
-        extension_name.to_camel_case().strip_prefix("Vk").unwrap()
+        extension_name.to_lower_camel_case().strip_prefix("Vk").unwrap()
```

Comment:
Lower... So this won't strip the uppercase Vk prefix if the whole thing is lowered or will it retain characters that were uppercase?

Please run the generator at least once to make sure output doesn't change or remains what we expect.